zip-####.rst

```diff
@@ -130,8 +181,8 @@ implement. In particular, it should not assume the creation of new mechanisms
 (e.g., election systems) or entities (for governance or development) for its
 execution; but it should strive to support and use these once they are built.
 
-Comply with legal, regulatory, and taxation constraints in pertinent
-jurisdictions.
+New Dev Fund recipients should comply with legal, regulatory, and taxation
```

Comment:
This should not be restricted to only new dev fund recipients.

Reply:
Correct, updated.  I had been mistakenly using  "New Dev Fund" to describe the entire concept of a post-halving new dev fund model, but in this case it creates confusion.